e2e/update_snapshots.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+function update_snapshot_e2e_test {
```

Comment:
nit: the name of this file makes it seem like it updates the snapshots - should probably have _test suffix

Reply:
✅